app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
The original concern I raised was that in preemptively checking the file type, we're bypassing our expected execution path of that method, which is that the value for the content type header is set by the mime type present in the technical metadata. This provides a promise, of sorts, that the kind of file we tell people we're sending is indeed what they're going to get. Only if that fails (there is no mimetype), do we fall back on the default `octet-stream`.

So I definitely agree with the statement "for all .jpg file we want it set as Content-Type: image/jpeg" but not with "regardless of how it is defined in the metadata". We don't want our metadata telling a different story than what we're telling downstream users. I see how that's kind of annoying in the case where an image renders just fine (the example in the referenced ticket), but it might not be the case in other instances, and we've had historical issues in that context, which I think is what @lsitu was getting at.